spec/features/deck_management_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe 'Management decks' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.